fakeprovide

```diff
@@ -66,7 +66,7 @@ Summary:	${RPM_SUMMARY:-Fake provide for $provide.}
 
 Group:		Fake
 License:	GPL
-BuildRoot:	%(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
+BuildRoot:	%{_topdir}/TEST
```

Comment:
Can you explain why this particular change was necessary? It's been a long time since I've had to fiddle with RHEL 5. Does rpm under RHEL 5 not respect the Buildroot setting? Or does it not correctly run the `mktemp ...` command?